Aliases/br

```diff
@@ -0,0 +1 @@
+../Formula/brooklyn-client.rb
```

Comment:
I don't think this is as widely used as they are. `aws` is pretty unambiguous at this point.